server_apps/data_transfer/Downloads/DownloadFiles.pl

```diff
@@ -270,25 +270,6 @@
 
 close ABXPFISH;
 
-## ZFIN-5649 
-$wtXpatFishWithBackSlash = '<!--|ROOT_PATH|-->/server_apps/data_transfer/Downloads/downloadsStaging/wildtype-expression_fish2.txt';
```

Comment:
I don't see this file still be generated. Has this been replaced with the new ones?

Reply:
This code opens "wildtype-expression_fish2.txt", removes all slash characters ("\"), writes the results to "wildtype-expression_fish.txt", then removes "wildtype-expression_fish2.txt".  I found that when I changed the delimiter from '	' to E'\t' this was no longer necessary so we can just write directly to "wildtype-expression_fish.txt"

Reply:
Great!